docs/assistant/security-assistant.asciidoc

```diff
@@ -52,15 +52,39 @@ NOTE: Elastic can automatically anonymize event data that you provide to AI Assi
 [[set-up-ai-assistant]]
 == Set up AI Assistant
 
-You must complete these steps before you can use AI Assistant:
+You must create a generative AI connector before you can use AI Assistant. 
 
-. Create an API key with your AI provider to authenticate requests from AI Assistant. You'll use this in the next step. Refer to the provider's documentation for generating API keys:
-+
-* https://platform.openai.com/docs/api-reference[OpenAI]
-* https://learn.microsoft.com/en-us/azure/cognitive-services/openai/reference[Azure OpenAI Service]
+. Open AI Assistant *Cmd + ;* (or *Ctrl + ;* on Windows), and click **Connector** *->* *Add new Connector**
+. Select either *Amazon Bedrock* or *OpenAI*. 
+. Enter the authentication details required for your chosen connector type, then click *Save*.
+
+For OpenAI and Azure OpenAI Service, you need to provide an API key. For Amazon Bedrock, you need to provide an access key and secret for an IAM user with at least the following permissions:
+
+.Expand permissions JSON
+[%collapsible]
+====
+```
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "VisualEditor0",
+            "Effect": "Allow",
+            "Action": [
+                "bedrock:InvokeModel",
+                "bedrock:InvokeModelWithResponseStream"
+            ],
+            "Resource": "*"
+        }
+    ]
+}
+```
+====
+
+For Amazon Bedrock, only Anthropic models are supported: `Claude`, and `Claude Instant`. You need to https://docs.aws.amazon.com/bedrock/latest/userguide/model-access.html#manage-model-access[enable them in AWS] before setting up an Amazon Bedrock connector. You can configure an Amazon Bedrock connector to use any https://docs.aws.amazon.com/general/latest/gr/bedrock.html[AWS region where Anthropic models are supported] by editing the *URL* field under *Connector settings*, for example by changing `us-west-2` to `eu-central-1`.
```

Comment:
The model names prob don't need formatting: monospace is typically for things the user types, and bold for clicking/selecting UI elements, and it doesn't look like the user is doing either here.

Also, if the user has to enable the models _before_ they set up a connector, we should tell them that _before_ the connector instructions.
```suggestion
For Amazon Bedrock, only Anthropic models are supported: Claude, and Claude Instant. You need to https://docs.aws.amazon.com/bedrock/latest/userguide/model-access.html#manage-model-access[enable them in AWS] before setting up an Amazon Bedrock connector. You can configure an Amazon Bedrock connector to use any https://docs.aws.amazon.com/general/latest/gr/bedrock.html[AWS region where Anthropic models are supported] by editing the *URL* field under *Connector settings*, for example by changing `us-west-2` to `eu-central-1`.
```